### PR TITLE
MessageX class removal

### PR DESCRIPTION
**Pull request by 3061LRTAGSPKJMORMRT**
- Removed the usage of the MessageEmbed, MessageButton, MessageSelectMenu, MessageActionRow class
- Discord.js v14 has renamed these, and it will be more work then, so why not start slowly now
- Moreover, we will have to require builders when doing so. Using objects is way better
- These classes return an object, so I converted it
- Please note that for embed colors, they will take a `Number` instead of a `hex string`.
- Use the `resolveColor` method that is imported from the `Util` class to convert names or the `RANDOM` color
- For components, they will have a `type` property
   * Type `1` is for Action Rows
   * Type `2` is for Buttons
   * Type `3` is for Select Menus

### CODE REVIEW

**Review by thewilloftheshadow — DISMISSED** [review]
This needs a [rebase](https://docs.github.com/en/get-started/using-git/about-git-rebase)

**Comment by 3061LRTAGSPKJMORMRT** [comment]
> This needs a [rebase](https://docs.github.com/en/get-started/using-git/about-git-rebase)

How do I do a rebase on a fork tho

**Comment by Hackante** [comment]
maybe `git rebase <Iink of repo>`

**Comment by 3061LRTAGSPKJMORMRT** [comment]
> maybe `git rebase <Iink of repo>`

It keeps saying it's up to date

**Comment by 3061LRTAGSPKJMORMRT** [comment]
Okay i ___think___ i did it.

**Review by Hackante — CHANGES_REQUESTED** [review]
There are more embeds in 
- ./server.js:248
- ./commands/bot/roleinfo.js:10
- ./commands/game/flee.js:27 ,  ./commands/game/flee.js:41

File: commands/bot/botinfo.js [hunk 1]
```diff
@@ -1,36 +1,18 @@
...
-`
-        )
+        embed.fields.push({
+            name: "**__Credits:__**",
+            value: `Bot Host, Retired Lead Developer: <@439223656200273932>\n` + `Lead Developer: Hackante#1482\n` + `Originally created by: Ashish#0540\n` + `Beta Testers: <@!159717599993790464> <@!719564153072910407> <@!801726595378315264> <@!406412325973786624> <@!620964363729371137> <@!452632560795648000>\n` + `Economy Designers: <@!801726595378315264> <@!406412325973786624>\n\n` + `__Special thanks to the following:__\n` + `<@!719564153072910407> for helping with many testing games, bug finding, organizing suggestions, and testing everything.\n` + `<@!620964363729371137> for organizing basically everything about the bot and GitHub, starting the bug tracker, keeping everything neat.\n` + `<@!263472056753061889> for stepping up everywhere and helping without me asking, both in the bot and on the tracker.\n\n` + `Thank you to all the Staff who come up with new ideas for the bot, and to everyone who has joined a Testes Gae or a Beta Testes game.\n` + `Thank you to all the Translators who provide the ability for us to be multilingual, both in the server and in our games.`,
```
[the remark this answers is not in the document]
Comment (thewilloftheshadow) [reply commands/bot/botinfo.js:14]: Removed what? Now you have staff on there twice

File: server.js [hunk 2]
```diff
@@ -245,19 +249,10 @@ client.on("ready", async () => {
...
-**Cost:** ${lot.cost}
-**Total Tickets:** ${lot.ticketsBought}
-**End Date:** <t:${Math.floor(lot.endDate / 1000)}:f>
-**Message ID:** ${lot.msg}
-
-**Participants:**
-${part.join(",\n")}`,
+                            {
+                                description: `**Pot:** ${lot.pot}\n` + `**Max Tickets:** ${lot.max}\n` + `**Cost:** ${lot.cost}\n` + `**Total Tickets:** ${lot.ticketsBought}\n` + `**End Date:** <t:${Math.floor(lot.endDate / 1000)}:f>\n` + `**Message ID:** ${lot.msg}\n\n` + `**Participants:**\n` + `${part.join(",\n")}`,
                                 color: "GREEN",
```
Comment (Hackante) [inline server.js:254]:
```suggestion
                                color: 0x00ff00,
```

**Review by Hackante — CHANGES_REQUESTED** [review]
Still two embeds in flee.js
then you are good to go ig

File: commands/game/flee.js [hunk 3]
```diff
@@ -38,16 +37,15 @@ module.exports = {
...
-**Warn ID:** ${warn.index}`,
-                            color: "DARK_RED",
+                            description: `You have received a gamewarn in ${message.guild.name}!\n` +
+                                `**Reason:** ${warn.reason}\n` +
+                                `**Gamecode:** ${warn.gamecode}\n` +
+                                `**Date:** <t:${(Date.now() / 1000).toFixed()}:f>\n\n` +
+                                `If you think this gamewarn was given by accident please [open a ticket](https://discord.com/channels/465795320526274561/606230556832825481/905800163069665280) in [#${client.channels.resolve("606230556832825481").name}](https://discord.com/channels/465795320526274561/606230556832825481).\n` +
+                                `**Warn ID:** ${warn.index}`,
+                            color: 0x992d22,
                         })
```
Comment (Hackante) [inline commands/game/flee.js:49]:
```suggestion
                        }
```

**Review by Hackante — CHANGES_REQUESTED** [review]

**Review by Hackante — APPROVED** [review]

**Pull request merged**